controllers/vspheremachine_controller.go

```diff
@@ -274,7 +269,37 @@ func (r machineReconciler) reconcileDeleteVMPre7(ctx *context.MachineContext) er
 	return nil
 }
 
+func (r machineReconciler) findVMPre7(ctx *context.MachineContext) (*infrav1.VSphereVM, error) {
```

Comment:
Curious what makes this method "pre vSphere 7", can we add a comment to clarify?

Reply:
@akutz - IIRC we introducted this (xxxPre7, e.g. `reconcileNormalPre7`) in case there were different ways to deal with vms